includes/onnxruntime_customop.hpp

```diff
@@ -99,7 +99,7 @@ struct CustomOpApi {
 template <typename TOp, typename TKernel>
 struct CustomOpBase : OrtCustomOp {
   CustomOpBase() {
-    OrtCustomOp::version = ORT_API_VERSION;
+    OrtCustomOp::version = 10;  // The minimum ORT version supported
```

Comment:
is this comment in this link inaccurate?
https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h#L3665